pkg/apply/task/apply_task.go

```diff
@@ -90,7 +83,7 @@ func (a *ApplyTask) Start(taskContext *taskrunner.TaskContext) {
 		klog.V(2).Infof("apply task starting (%d objects)", len(objects))
 		// Create a new instance of the applyOptions interface and use it
 		// to apply the objects.
-		ao, dynamic, err := applyOptionsFactoryFunc(taskContext.EventChannel(),
+		ao, _, err := applyOptionsFactoryFunc(taskContext.EventChannel(),
```

Comment:
Do we still need to return a dynamic client from `applyOptionsFactoryFunc`?

Reply:
No--removed.